### PR TITLE
Add software tools built by SERT members

### PR DESCRIPTION
**Pull request by SUPERCILEX**
I wasn't sure where to put this, but I would like us to show what the software team has accomplished. For now, I just dumped it in the outreach page since our tools technically are.

### CODE REVIEW

File: _pages/outreach.md [hunk 1]
```diff
@@ -37,14 +37,25 @@ camp at the University of Oregon.
...
+SERT has built many tools over the years to make various tasks faster and easier, all of which are open-source.
+Here are just a few of them:
+
+1. **[🤖 Robot Scouter](https://github.com/SUPERCILEX/Robot-Scouter/)** is a simple yet powerful Android
+scouting app with customizable scouting templates and advanced spreadsheet exporting for painless data analysis.
+1. The **[🕑 Attendance Client](https://github.com/SouthEugeneRoboticsTeam/Attendance-Client)**
+is a cross-platform desktop app built with Firebase and React to make keeping track of team members' hours a piece of cake.
+1. Vision processing code for [single-board computers](https://github.com/SouthEugeneRoboticsTeam/vision)
+such as the Raspberry Pi or for the
+[roboRIO](https://github.com/SouthEugeneRoboticsTeam/Steamworks-2017/blob/71a63ba36a162f533b1fb9a52d1f1b8c61748378/src/org/usfirst/frc/team2521/robot/vision/Looper.java).
```
Comment (andrewda) [inline _pages/outreach.md]: Maybe make this the master branch instead of a specific blob? Doesn't really matter but would keep the code up to date.

Comment (SUPERCILEX) [inline _pages/outreach.md]: I specifically chose to make it a permalink so we won't have to remember to update the link if the looper got moved or something. The worst thing ever IMO is to click a dead link on GitHub.

Comment (andrewda) [inline _pages/outreach.md]: K I guess that makes sense

Comment (SUPERCILEX) [inline _pages/outreach.md]: Lol, lots of things are making sense to you 😆

**Pull request merged**